packages/near-fast-auth-signer/src/api/index.ts

```diff
@@ -18,10 +18,17 @@ export const fetchAccountIds = async (publicKey: string): Promise<string[]> => {
     throw new Error(`HTTP error! status: ${res.status}`);
   }
 
-  const accountIds = await res.json();
+  let accountIds: string[] = await res.json();
+
+  if (accountIds.length === 0) {
+    const accountId = await window.firestoreController.getAccountIdByPublicKey(publicKey);
+    accountIds =  accountId ? [accountId] : [];
+  }
+
   if (accountIds.length === 0) {
     throw new Error('Unable to retrieve account id');
```

Comment:
my assumption of using firebase is that if we use this, and ideally we will get 100% account Id return (for ones that exist), if it doesn't exist, instead of throw error, shouldn't we redirect them to verify email instead of throw error?

Reply:
I believe this was introduce by you on the PR for non-WebauthN browsers (at least that's what my git says, please correct if I'm wrong). We can change it, but I would suggest doing it on another PR as this change it's not related to the current PR. 

<img width="956" alt="Screenshot 2024-02-01 at 8 57 34 AM" src="https://github.com/near/fast-auth-signer/assets/22358004/7bd20ecd-d58d-4473-ae79-9e69e97b46a0">

Reply:
so I left this only because [file](https://github.com/near/fast-auth-signer/blob/105320cf734da0746de2ee0b14c306749fa3fe1f/packages/near-fast-auth-signer/src/api/index.ts#L15) wasn't part of this PR or wasn't expecting it to be coming. If we have this code, above can stay as it is